src/_igraph/graphobject.c

```diff
@@ -16563,6 +16622,24 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "  no guarantees are given about the ordering of edge IDs within cycles.\n"
    "@return: the cycle basis as a list of tuples containing edge IDs"
   },
+  {"simple_cycles", (PyCFunction) igraphmodule_Graph_simple_cycles,
+   METH_VARARGS | METH_KEYWORDS,
+   "simple_cycles(mode=None, min=0, max=-1, output=\"vpath\")\n--\n\n"
```

Comment:
```suggestion
   "simple_cycles(mode=None, min=-1, max=-1, output=\"vpath\")\n--\n\n"
```

Of course 0 works too, but using `-1` would make it easier to create consistent and predictable interfaces in the future.